x/wasm/types/test_fixtures.go

```diff
@@ -257,6 +257,25 @@ func MigrateContractProposalFixture(mutators ...func(p *MigrateContractProposal)
 	return p
 }
 
+func SudoContractProposalFixture(mutators ...func(p *SudoContractProposal)) *SudoContractProposal {
+	const (
+		contractAddr = "cosmos14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9s4hmalr"
+		anyAddress   = "cosmos1qyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqs2m6sx4"
```

Comment:
unused

Reply:
SudoContractProposalFixture is used in the test above?